com.unity.multiplayer.mlapi/Runtime/Core/NetworkedObject.cs

```diff
@@ -39,9 +38,10 @@ internal void ValidateHash()
         }
 
         /// <summary>
-        /// Gets the NetworkingManager that owns this NetworkedObject instance
+        /// The NetworkingManager that owns this Object. 
         /// </summary>
-        public NetworkingManager NetworkManager => NetworkingManager.Singleton;
+        public NetworkingManager NetManager { get; internal set; }
```

Comment:
Why did we change the name to NetManager ?

Reply:
yeah, we shouldn't change that like this — we're going to have a renaming RFC soon where we will specifically discuss naming types.

Reply:
these are all normalized to NetworkManager now.